x-pack/plugins/security_solution/public/overview/containers/overview_risky_host_links/use_risky_host_links.ts

```diff
@@ -96,13 +96,15 @@ export const useRiskyHostLinks = ({ to, from }: { to: string; from: string }) =>
   }, [addError, error, setIsModuleEnabled]);
 
   useEffect(() => {
-    start({
-      data,
-      timerange: { to, from, interval: '' },
-      defaultIndex: [RISKY_HOSTS_INDEX],
-      filterQuery: '',
+    spaces.getActiveSpace().then((space) => {
+      start({
+        data,
+        timerange: { to, from, interval: '' },
+        defaultIndex: [`${RISKY_HOSTS_INDEX_PREFIX}${space.id}`],
```

Comment:
Nit: Maybe this should be abstracted to a helper function if it's going to be reused (e.g. by the Alert details effort), e.g. `getHostRiskIndex: (spaceId: string) => string`